dev/publish_bom.py

```diff
@@ -122,13 +121,14 @@ def publish_changelog_gist(self):
       # Also add some identifying information.
       raw_content_lines[0] = spinnaker_version
       timestamp = '{:%Y-%m-%d %H:%M:%S}'.format(datetime.datetime.now())
-      signature = '\n\nGenerated by {0} at {1}'.format(socket.gethostname(), timestamp)
+      signature = '\n\nGenerated by at {0}'.format(timestamp)
```

Comment:
I think you should just replace socket.gethostname() with the logical user (e.g. spinnaker-release or organization).
If it is just a timestamp then s/by //. However I think there should be organizational traceability here.